GenderPayGap.WebUI/Services/EmailSendingService.cs

```diff
@@ -116,6 +116,21 @@ public static async void SendScopeChangeOutEmail(string emailAddress, string org
 
             await AddEmailToQueue(notifyEmail);
         }
+        public static async void SendCreateAccountPendingVerificationEmail(string emailAddress, string url)
+        {
+            var personalisation = new Dictionary<string, dynamic> {
+                {"url", url}, {"Environment", Config.IsProduction() ? "" : $"[{Config.EnvironmentName}] "}
+            };

```

Comment:
I'd prefer these two dictionary entries on separate lines (although I know the auto-formatted sometimes puts them on the same line!) - for a dictionary, I think it makes it clearer that we're adding two entries

i.e.
```csharp
var personalisation = new Dictionary<string, dynamic> {
    {"url", url},
    {"Environment", Config.IsProduction() ? "" : $"[{Config.EnvironmentName}] "}
};
```